datafusion/benches/filter_query_sql.rs

```diff
@@ -31,7 +31,7 @@ async fn query(ctx: &mut ExecutionContext, sql: &str) {
     let rt = Runtime::new().unwrap();
 
     // execute the query
-    let df = ctx.sql(sql).unwrap();
+    let df = rt.block_on(ctx.sql(sql)).unwrap();
```

Comment:
Does this change mean the benchmark time will now include the planning time (whereas before it did not)?

Reply:
This shouldn't change anything, to my knowledge `rt.block_on` is independent of Criterion. Also, `sql()` is only blocking for `CreateExternalTable` queries, so for normal queries it will actually never yield back to the runtime.